src/synthesizer_grids/incident/sps/install_maraston13.py

```
@@ -129,10 +119,20 @@ def make_grid(model, imf, input_dir, grid_dir):
     if not os.path.exists(input_dir):
         os.mkdir(input_dir)
 
-    # run the single kroupa100 model
-    imf = "kroupa100"
-    make_grid(model, imf, input_dir, grid_dir)
-
     # then run the rest
     for imf in imfs:
-        make_grid(model, imf, input_dir, grid_dir)
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)